lib/liquid/context.rb

```diff
@@ -47,6 +48,22 @@ def add_filters(filters)
       @strainer = nil
     end
 
+    def add_global_filters(global_filters)
+      gf = [global_filters].flatten.compact
+
+      gf.each do |p|
+        if p.respond_to?(:call)
```

Comment:
what's this check for? why would you call `add_global_filters` with something that doesn't respond to `call`?